### PR TITLE
Vault total value

### PR DESCRIPTION
**Pull request by Pawel-Szydlo**

### CODE REVIEW

**Review by PawelKrawczyk96 — APPROVED** [review]

File: src/components/Statistics/StatisticCardAll.tsx [hunk 1]
```diff
@@ -33,7 +39,7 @@ export const StatisticCardAll: React.FC<IProp> = ({ data, debtCurrent, collatera
         <Grid id='debt' item xs={12} sm={5}>
           <StatisticsCard
             name='Debt'
-            value={debtCurrent.reduce((a, b) => a + b.value, 0)}
+            value={debtCurrent.reduce((a, b) => a + b.value, 0) + debtVault}
```
Comment (awojciak) [inline src/components/Statistics/StatisticCardAll.tsx:42]: I talked about that with Norbert and he said that nothing should be added to value of total debt (which is the value of minted synthetics and it already grows when you mint something through staking or swapline)

**Review by awojciak — CHANGES_REQUESTED** [review]
I left one comment, check this out

File: src/components/Statistics/StatisticCardAll.tsx (hunk 1 above)
Comment (Pawel-Szydlo) [reply src/components/Statistics/StatisticCardAll.tsx:42]: ok, I'll delete it

**Review by awojciak — CHANGES_REQUESTED** [review]
obraz
fix this

**Comment by Pawel-Szydlo** [comment]
This?  So what exactly?

**Review by wojciech-cichocki — COMMENTED** [review]
If the problem is two vault entry for sny/xsol. There are multiple vault type for the same pair synthetic/collateral, but with different c-ratio and interest rate

**Review by awojciak — APPROVED** [review]

**Pull request merged**